src/gists/gist.jl

```diff
@@ -0,0 +1,79 @@
+type Gist <: GitHubType
+    url::Nullable{HttpCommon.URI}
+    forks_url::Nullable{HttpCommon.URI}
+    commits_url::Nullable{HttpCommon.URI}
+    id::Nullable{GitHubString}
+    description::Nullable{GitHubString}
+    public::Nullable{Bool}
+    owner::Nullable{Owner}
+    user::Nullable{Owner}
+    truncated::Nullable{Bool}
+    comments::Nullable{Int}
+    comments_url::Nullable{HttpCommon.URI}
+    html_url::Nullable{HttpCommon.URI}
+    git_pull_url::Nullable{HttpCommon.URI}
+    git_push_url::Nullable{HttpCommon.URI}
+    created_at::Nullable{Dates.DateTime}
+    updated_at::Nullable{Dates.DateTime}
+    forks::Nullable{Vector{Gist}}
+    files::Nullable{Dict}
+    history::Nullable{Vector{Dict}}
```

Comment:
Yeah but it is not `Owner` or even `Vector{Owner}` here. The API is:

```
"history": [
    {
      "url": "https://api.github.com/gists/aa5a315d61ae9438b18d/57a7f021a713b1c5a6a199b54cc514735d2d462f",
      "version": "57a7f021a713b1c5a6a199b54cc514735d2d462f",
      "user": {
        "login": "octocat",
        "id": 1,
        [...]
      },
      "change_status": {
        "deletions": 0,
        "additions": 180,
        "total": 180
      },
      "committed_at": "2010-04-14T02:15:15Z"
    }
  ]
```

So we have a Vector of `Dicts` where one of the `value` in each `Dict` is an `Owner`. I don't see a way of specifying this in the types.